packages/packages/site-settings/src/components/ported-primary-action.tsx

```diff
@@ -0,0 +1,10 @@
+import Portal from './portal';
+import PrimaryAction from './primary-action';
+
+export default function PortedPrimaryAction() {
```

Comment:
```suggestion
export default function PortalledPrimaryAction() {
```
also in other places, and also the file name

Reply:
Done